### DISCUSSION TITLE
How to use with UI Toolkit

### DISCUSSION THREAD

**Discussion by DeepP2667** [KyryloKuzyk/PrimeTween#107] [FAQ] [answered]
Hi, I am new to using this library. I was wondering what function would I use to move VisualElements in UI Toolkit? It seems like I can use Custom function but was wondering if there were more UI toolkit specific functions.
Thanks!

**Comment by KyryloKuzyk** [comment, marked as answer]
Hi! You can animate the UI Toolkit element by typing `Tween.VisualElement...` and your IDE will suggest all currently supported animation types. 

Also, you can easily write custom animations if the built-in animations are not enough for you: https://discussions.unity.com/t/primetween-high-performance-animations-and-sequences/926420/313

**Reply by DeepP2667** [reply]
Thank you! I think 1 example using VisualElement in the docs would be nice as someone else in that thread had the same idea as me, using Tween.Custom. But overall, thanks for a great and smooth package!
